src/components/shared/Entity/DetailsForm.tsx

```diff
@@ -17,45 +22,83 @@ import {
     defaultRenderers,
     showValidation,
 } from 'services/jsonforms';
-import { entityCreateStoreSelectors } from 'stores/Create';
+import { Details, DetailsFormState } from 'stores/DetailsForm';
+import { EndpointConfigState } from 'stores/EndpointConfig';
+import { EntityFormState } from 'stores/FormState';
 import { Grants } from 'types';
 
 interface Props {
     connectorTags: ConnectorWithTagDetailQuery[];
     accessGrants: Grants[];
     draftEditorStoreName: DraftEditorStoreNames;
+    formStateStoreName: FormStateStoreNames;
+    detailsFormStoreName: DetailsFormStoreNames;
+    endpointConfigStoreName: EndpointConfigStoreNames;
 }
 
 function DetailsForm({
     connectorTags,
     accessGrants,
     draftEditorStoreName,
+    formStateStoreName,
+    detailsFormStoreName,
+    endpointConfigStoreName,
 }: Props) {
     const intl = useIntl();
     const [searchParams] = useSearchParams();
-    const connectorID = searchParams.get(
-        authenticatedRoutes.captures.create.params.connectorID
-    );
+    const connectorID =
```

Comment:
The materialization buttons on the connectors table never worked, thus I am enabling them here. This was _quick_ logic thrown in during testing to remove the defect.